sdkms/support.go

```diff
@@ -310,3 +311,119 @@ const (
 )
 
 func someString(val string) *string { return &val }
+
+type HyperHttpMthod string
+
+// Common HTTP methods.
+//
+// Unless otherwise noted, these are defined in RFC 7231 section 4.3.
+const (
+	MethodGet     HyperHttpMthod = "GET"
+	MethodHead    HyperHttpMthod = "HEAD"
+	MethodPost    HyperHttpMthod = "POST"
+	MethodPut     HyperHttpMthod = "PUT"
+	MethodPatch   HyperHttpMthod = "PATCH" // RFC 5789
+	MethodDelete  HyperHttpMthod = "DELETE"
+	MethodConnect HyperHttpMthod = "CONNECT"
+	MethodOptions HyperHttpMthod = "OPTIONS"
+	MethodTrace   HyperHttpMthod = "TRACE"
+)
+
+type RemovableInterface interface {
+}
+
+func ParseCustomMetadata(v map[string][]string, cus map[string]string) {
```

Comment:
This function is not named properly, it's not parsing anything. Also is this meant to be public?

Reply:
function not needed